doc/reference/reference_sql/sql-features.rst

```diff
@@ -38,7 +38,7 @@ marked "Okay" will probably be balanced by tests which are unfairly marked "Fail
     |            |                                               |                                                          | columns (created from Lua) from FLOAT/DOUBLE/REAL ones  |
     |            |                                               |                                                          | (created from SQL).                                     |
     +------------+-----------------------------------------------+----------------------------------------------------------+---------------------------------------------------------+
-    | E011-03    | DECIMAL and NUMERIC data types                | ``CREATE TABLE td (s1 NUMERIC PRIMARY KEY);``            | Fail, DECIMAL and NUMERIC data types are not supported  |
+    | E011-03    | DECIMAL and NUMERIC data types                | ``CREATE TABLE td (s1 NUMERIC PRIMARY KEY);``            | Fail, NUMERIC data types are not supported              |
```

Comment:
Minor: Is DECIMAL left intentionally in the second column?

Reply:
Yes, that is the official Feature Name for feature E011-03 in the SQL:2016/Foundation document, so we have to say it that way.